src/components/Form.tsx

```
@@ -1,48 +1,77 @@
+        {children}
+      </form>
     </FormProvider>
   );
 }
 
-// Error 컴포넌트
-type ErrorProps<T extends FieldValues> = {
-  name: Path<T>;
-};
-
 function Error<T extends FieldValues>({ name }: ErrorProps<T>) {
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)